Lib/test/test_eagerly_import.py

```diff
@@ -0,0 +1,82 @@
+'''
+- Target:
+    Add a new sample test that checks Python imports are executed eagerly by default.
+
+- Description:
+    We need to confirm one of these two options based on the scenario. Otherwise, this test fails.
+        - If it is in default, the imports are executed eagerly.
+        - If Lazy Imports is enabled, the imports are NOT executed eagerly.
+'''
+
+import os
+import sys
+import unittest
+# import json
+import sqlite3
+
+class LazyImportsEnabledChecker:
+    def __init__(self):
+        self.enabled = self.check_lazy_imports()
+
+    # confirm if Lazy Imports is enabled or not
+    def check_lazy_imports(self):
+        # lazy imports is not enabled in default
+        enabled = False
+
+        # check if Lazy Imports is turned on
+        enabled |= self.check_flag()
+        enabled |= self.check_env_var()
+        return enabled
+
+    # check if the env variable PYTHONLAZYIMPORTSALL is set
+    def check_env_var(self):
+        env_variables = os.environ
+        if "PYTHONLAZYIMPORTSALL" not in env_variables:
+            return False
+
+        # check this setting of this env_var
+        if env_variables["PYTHONLAZYIMPORTSALL"] == "0":
+            return False
+        else:
+            return True
+
+    # check if -L flag is added to the Python interpreter
+    def check_flag(self):
+        argvs = sys.argv
+        for argv in argvs:
+            if argv == "-L" or argv == "-l":
```

Comment:
Only uppercase `-L` enables lazy imports. Most likely, `-l` would result in an `Unknown option` error.